scripts/build.sh

```diff
@@ -1,4 +1,5 @@
 #!/usr/bin/env bash
 rm -rf dist
```

Comment:
I can't remember if we talked about this.  Do we want to run the linting/tests as part of the build script either by direct invocation here or via a pre-build script calling `npm run tests tests`?  I see that it's already manually run as part of the proper build, so, I could see us keeping it composable and just opting into it when needed.  I also see it as a pre-push hook, so, it might not be strictly necessary here.